cpp/ql/src/semmle/code/cpp/ir/implementation/TempVariableTag.qll

```diff
@@ -1,6 +1,15 @@
+/**
+ * Defines the public interface to temporary variable tags, which describe the reason a particular
+ * `IRTempVariable` was generated.
+ */
 private import internal.TempVariableTagInternal
 private import Imports::TempVariableTag
 
+/**
+ * Describes the reason that a particular IR temporary variable was generated. For example, it could
```

Comment:
QLDoc for a class is usually a singular noun phrase. I'm not clear on whether that's a style guide requirement or not. All the examples use the "An expression..." style, but it's not explicitly written.

Reply:
```suggestion
 * A reason that a particular IR temporary variable was generated. For example, it could
```

Reply:
Yes, we use A/An. See https://wiki.semmle.com/pages/viewpage.action?pageId=74613282#QLDocStyleGuide-QLDocforclasses (linked from https://github.com/github/codeql-c-analysis-team/issues/40). That part of the style guide hasn't made it to the public repo yet,.

Reply:
Fixed.